prometheus-exporters/statsd-exporter/values.yaml

```diff
@@ -1,98 +1,78 @@
-  #   cpu: 100m
-  #   memory: 128Mi
-
-web:
-  # The address on which to expose the web interface and generated Prometheus metrics.
-  port: 9102
-
-  # Path under which to expose metrics.
-  path: /metrics
-
-statsd:
-  # The UDP port on which to receive statsd metric lines.
-  udpPort: 9125
-
-  # The TCP port on which to receive statsd metric lines.
-  tcpPort: 9125
-
-  # Maximum size of your metric mapping cache.
-  # Relies on least recently used replacement policy if max size is reached.
-  cacheSize: 1000
-
-  # Size of internal queue for processing events.
-  eventQueueSize: 10000
-
-  # Number of events to hold in queue before flushing.
-  eventFlushThreshold: 1000
-
-  # Time interval before flushing events in queue.
-  eventFlushInterval: 200ms
-
-  # Metric mapping configuration
-  # Reference for this file's format:
-  # <https://github.com/prometheus/statsd_exporter#metric-mapping-and-configuration>
-  mappingConfig: |-
-    defaults:
-      ttl: 1h 
-
-owner-info:
-  support-group: observability
-  maintainers:
-    - Thomas Graichen
-    - Martin Vossen
-  helm-chart-url: https://github.com/sapcc/helm-charts/tree/master/prometheus-exporters/statsd-exporter
\ No newline at end of file
+  # Default values for prometheus-statsd-exporter.
+  # This is a YAML-formatted file.
+  # Declare variables to be passed into your templates.
+  image:
+    repository: keppel.eu-de-1.cloud.sap/ccloud-dockerhub-mirror/prom/statsd-exporter
+    tag: v0.23.1
+    pullPolicy: IfNotPresent
+
+  replicaCount: 1
+  nameSpace: cc3test
+
+  ingress:
+    enabled: true
+    tls: true
+    disco: true
+  #  no_keep_alive: true
+
+  exporters:
+    - name: 'statsd-exporter'
+      enabled: true
+      udpPort: 9125 # The UDP port on which to receive statsd metric lines.
+      tcpPort: 9125 # The TCP port on which to receive statsd metric lines.
+      cacheSize: 1000 # Maximum size of your metric mapping cache.
+      eventQueueSize: 10000 # Size of internal queue for processing events.
+      eventFlushThreshold: 1000 # Number of events to hold in queue before flushing.
+      eventFlushInterval: 200ms # Time interval before flushing events in queue.
+      # Metric mapping configuration (see https://github.com/prometheus/statsd_exporter#metric-mapping-and-configuration for details
+      mappings: |-
+        defaults:
+          ttl: 1h # metrics expire after ttl
+      # Resources
+      resources:
+        requests:
+          cpu: 200m
+          memory: 200Mi
+        # limits:
+        #   cpu: 100m
+        #   memory: 128Mi
+      # Prometheus
+      metricsPort: 9102 # The address on which to expose the web interface and generated Prometheus metrics.
+      metricsPath: /metrics # Path under which to expose metrics.
+      prometheus: infra-frontend
+      scrapeInterval: 60s
+      scrapeTimeout: 55s
+
+    - name: 'statsd-exporter-inf'
+      enabled: true
+      udpPort: 9125 # The UDP port on which to receive statsd metric lines.
```

Comment:
just a thought: would it maybe make sense to use a different port here? but maybe it does not make too much sense ...

Reply:
Doesn't matter since each has it's own ingress,